controllers/workspace/finalize.go

```diff
@@ -17,15 +17,15 @@ import (
 
 	dw "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
 
-	"github.com/devfile/devworkspace-operator/controllers/workspace/provision"
-	"github.com/devfile/devworkspace-operator/pkg/provision/storage"
-
 	"github.com/go-logr/logr"
 	coputil "github.com/redhat-cop/operator-utils/pkg/util"
 	corev1 "k8s.io/api/core/v1"
 	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/devfile/devworkspace-operator/pkg/provision/storage"
+	provision "github.com/devfile/devworkspace-operator/pkg/provision/workspace"
```

Comment:
really minor. Maybe it makes sense to drop provision alias, or make it wsProvision to avoid confusion with `provision/storage`...

Reply:
Yeah, I had a bit of trouble with this since we're having some aliasing issues :smile: 

- Can't use `workspace` as we name the DevWorkspace instance that
- `provision` isn't ideal since it's completely unrelated to the package (as you point out)
- `workspaceprovision` is too long

I think `wsprovision` makes sense. I'll update the PR -- do we want to settle on `wsProvision` or `wsprovision`? We kind of use both (`k8serrors` vs `k8sErrors`) but I think the standard is all-lowercase.

Reply:
I'm ok with any